### ISSUE TITLE
FTB Quest Importer

### ISSUE THREAD

**Issue by FeedCat02** [Funwayguy/StandardQuestingPack#106]
FTB uses .snbt files instead of .nbt files so the importer doesn't work anymore

**Comment by Funwayguy** [comment]
Yes. They retroactively changed all the formats to their own SNBT format (non-compliant to 1.12.2) among other strange modifications. I've already written a tool to deal with this but I'll need a zip of the intact FTBQ directories to convert it over.

**Comment by FeedCat02** [comment]
I personally didnt have too many quests to move over my quests manually.  Just thought I would report the issue incase other people needed this because of how ftb quests doesnt support 1.12 anymore.
